### DISCUSSION TITLE
[Auth][Mobile] Open Keyboard automatically when "Focus Search on app start" is enabled

### DISCUSSION THREAD

**Discussion by gokulkgm** [ente-io/ente#1967] [Enhancements]
Hello,

I activated the "Focus search on app start" feature, and now each time I launch the app, I must tap the search bar to bring up the keyboard. Therefore, an option to automatically open the keyboard would be greatly beneficial.

**Comment by klaas0** [comment]
There is already an issue created for this: https://github.com/ente-io/ente/issues/279
